charts/logging-apps/values.yaml

```diff
@@ -53,7 +53,7 @@ fluentd:
   # fluentd.chart -- Chart
   chart: "fluentd"
   # fluentd.targetRevision -- [fluentd Helm chart](https://github.com/bitnami/charts/tree/master/bitnami/fluentd) version
-  targetRevision: "3.0.*"
+  targetRevision: "3.2.*"
```

Comment:
Introduction of `bitnami/common` (library dependency) https://github.com/bitnami/charts/tree/master/bitnami/fluentd#to-310